packages/web/src/components/gcds-search/stories/overview.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, Canvas, Story } from '@storybook/blocks';
+import * as Search from './gcds-search.stories';
+
+<Meta of={Search} name="Overview" />
+
+# Search<br/>`<gcds-search>`
+
+_Also called: error, inline error._
+
+An error message is a description of a problem blocking a user goal.
+
+<Canvas
+  of={Search.Default}
+  story={{ inline: true }}
+/>
+
+## Examples
+<br/>
+
+### Default
```

Comment:
Can we add a few more examples?

For example, what does the default look like in French? Stepper has language examples:
https://cds-snc.github.io/gcds-components/?path=/docs/components-stepper--overview